cmd/meroxa/global/metrics_test.go

```diff
@@ -93,7 +93,7 @@ func TestAddError(t *testing.T) {
 	}
 
 	err := "unexpected error"
-	addError(event, fmt.Errorf(err))
+	addError(event, fmt.Errorf("%s", err))
```

Comment:
Wouldn't this do the trick?
```suggestion
	addError(event, err)
```

Reply:
Yes, this reads great -- thank you!

Reply:
Woops, it seems that will fail the tests:

```
  Error: cannot use err (variable of type string) as error value in argument to addError: missing method Error (typecheck)
  
  Error: issues found
```

Should we use the `addError(event, fmt.Errorf("%s", err))` syntax instead? What do you think?